src/status_im/utils/ethereum/eip681.cljs

```diff
@@ -23,21 +23,22 @@
 (def key-value-format (str "([^" parameter-separator key-value-separator "]+)"))
 (def query-pattern (re-pattern (str key-value-format key-value-separator key-value-format)))
 
-(def valid-native-arguments #{:value :gas})
+(def valid-native-arguments #{:value :gas :gasPrice})
 
 (defn- parse-query [s]
   (into {} (for [[_ k v] (re-seq query-pattern (or s ""))]
              [(keyword k) v])))
 
 (defn- parse-native-arguments [m]
-  (when (set/superset? valid-native-arguments (set (keys m)))
```

Comment:
Can you explain implementation change here? Seems different but don't quite get

Reply:
Invalid parameters used to reject the whole URI now invalid arguments are ignored. This ensures we can digest URI that might contain extra parameters that we don't support.